paymentsheet/src/test/java/com/stripe/android/customersheet/data/DefaultCustomerSessionElementsSessionManagerTest.kt

```diff
@@ -0,0 +1,276 @@
+package com.stripe.android.customersheet.data
+
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.customersheet.CustomerSheet
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+import com.stripe.android.customersheet.utils.FakeCustomerSessionProvider
+import com.stripe.android.isInstanceOf
+import com.stripe.android.model.ElementsSession
+import com.stripe.android.paymentsheet.ExperimentalCustomerSessionApi
+import com.stripe.android.paymentsheet.FakePrefsRepository
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.model.SavedSelection
+import com.stripe.android.paymentsheet.repositories.ElementsSessionRepository
+import com.stripe.android.testing.PaymentIntentFactory
+import com.stripe.android.utils.FakeElementsSessionRepository
+import kotlinx.coroutines.test.runTest
+import org.junit.Test
+import kotlin.coroutines.coroutineContext
+
+@OptIn(ExperimentalCustomerSheetApi::class, ExperimentalCustomerSessionApi::class)
+class DefaultCustomerSessionElementsSessionManagerTest {
+    @Test
+    fun `on fetch elements session, should set parameters properly`() = runTest {
```

Comment:
should this also test the side effect of setting the cached customer ephemeral key? (or maybe that should be done in a separate test)